### DISCUSSION TITLE
How to manage multiple environments Dev and Prod?

### DISCUSSION THREAD

**Discussion by ghadeeraqraa1992** [invertase/react-native-firebase#5136] [Q&A] [answered]
How to manage multiple environments Dev and Prod for Firebase with React Native App using only one project on firebase console ?

**Comment by mikehardy** [comment, marked as answer]
There are an infinite number of answers to that question but the short answer is do not do that ever

Why? You will be mixing your dev data with your prod data. That's not a good practice.

Have a dev firebase project. Have a prod firebase project.

Then use one of the infinite ways to select the right one as you build - I personally use android flavors and iOS targets where the config files on android are in separate flavor filesystem trees, and on iOS the correct firebase project plist files are included via file membership in the targets.

**Reply by andersonaddo** [reply]
https://github.com/invertase/react-native-firebase/issues/3504
